esmvalcore/cmor/_fixes/cmip5/hadgem2_cc.py

```diff
@@ -1,34 +1,10 @@
-
 """Fix HadGEM2_CC."""
-import numpy as np
-
 from ..fix import Fix
+from .hadgem2_es import AllVars as BaseAllVars
 
 
-class AllVars(Fix):
-    """Fix common errors for all vars."""
-
-    def fix_metadata(self, cubes):
-        """
-        Fix latitude.
-
-        Parameters
-        ----------
-        cube: iris.cube.CubeList
-
-        Returns
-        -------
-        iris.cube.CubeList
-
-        """
-        for cube in cubes:
-            lats = cube.coords('latitude')
-            if lats:
-                lat = cube.coord('latitude')
-                lat.points = np.clip(lat.points, -90., 90.)
-                lat.bounds = np.clip(lat.bounds, -90., 90.)
-
-        return cubes
+class AllVars(BaseAllVars):
+    """Fix errors common to all vars."""
```

Comment:
```suggestion
AllVars = BaseAllVars
```

Reply:
Not too sure about this one, why is it better to write it like this?

Reply:
Anyway, is be happy to change it, just curious

Reply:
It's shorter 😁 Both versions are fine for me, your call